src/elements/content-sidebar/versions/VersionsSidebar.js

```diff
@@ -49,8 +61,26 @@ const VersionsSidebar = ({ error, isLoading, parentName, versions, ...rest }: Pr
         >
             <LoadingIndicatorWrapper className="bcs-Versions-content" crawlerPosition="top" isLoading={isLoading}>
                 {showError && (
-                    <InlineError title={<FormattedMessage {...messages.versionServerError} />}>
+                    <InlineError
+                        title={
+                            errorUpsell ? (
```

Comment:
I think this inline error can be used for other API call failures. It also doesn't seem like `error` and `errorUpsell` have much in common. Can set `error` to undefined in the container and pass in a `showUpgrade` prop, instead?

Reply:
We could also consider passing in `errorTitle` from the container, which would simplify this quite a bit.

Reply:
I added an errorTitle, and cleaned up variable names, I will be using the showUpgrade approach with the other requirements, as those will no longer be errors from the API.